api/internal/target/kusttarget_configplugin.go

```diff
@@ -58,7 +58,9 @@ func (kt *KustTarget) configureBuiltinTransformers(
 		builtinhelpers.ReplicaCountTransformer,
 		builtinhelpers.ImageTagTransformer,
 		builtinhelpers.ReplacementTransformer,
-	} {
+		builtinhelpers.SortOrderTransformer,
```

Comment:
I see you've put the `SortOrderTransformer` at the end of every list here, can you confirm that it does indeed run after all the other transformers? I'm not 100% sure which list determines the transformer order.

Reply:
I don't think we need this either. Since we decided to call the transformer explicitly in kustomizer.go, it should never be implicitly ordered against the other plugins via this list.